views/layouts/boilerplate.ejs

```diff
@@ -5,36 +5,20 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>PIXELIFY</title>
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <link
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css"
-      rel="stylesheet"
-      integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT"
-      crossorigin="anonymous"
-    />
     <link rel="stylesheet" href="/stylesheets/navbar.css" />
-    <link rel="stylesheet" href="/stylesheets/cards.css" />
+    <link rel="stylesheet" href="/stylesheets/footer.css" />
+    <%if(extraStyles !== "undefined"){%> <%-extraStyles%><%}%>
```

Comment:
I would recommend to check `if (extraStyles)` otherwise if you pass an empty string it will pass the check. By doing a truthy check it will save you from such cases where you accidentally pass a wrong value. 

You can read about truthy and falsy values here: 
https://developer.mozilla.org/en-US/docs/Glossary/Truthy 
https://developer.mozilla.org/en-US/docs/Glossary/Falsy